vpr/src/place/feasible_region_move_generator.h

```diff
@@ -0,0 +1,10 @@
+#ifndef VPR_FEASIBLE_REGION_MOVE_GEN_H
+#define VPR_FEASIBLE_REGION_MOVE_GEN_H
+#include "move_generator.h"
+#include "timing_place.h"
+
+class FeasibleRegionMoveGenerator : public MoveGenerator {
```

Comment:
Comment what this move does, and include a reference to a paper that explains why it works.

Reply:
Done